### ISSUE TITLE
`%error` does not abort specfile processing when called through lua

### ISSUE THREAD

**Issue by gotmax23** [rpm-software-management/rpm#2653]
Take the following macro definitions:

``` spec
%error_1 %{error:Sorry, this is an error}
%error_2 %{lua:
    rpm.expand("%{error:Sorry, this is an error}")
}
%error_3 %{lua:
    macros.error({"Sorry, this is an error"})
}
```

``` console
$ rpm -E %error_1
error: Sorry, this is an error
$ echo $?
1
$ rpm -E %error_2
error: Sorry, this is an error
error: lua script failed: [string "<lua>"]:2: error expanding macro
$ echo $?
1
$ rpm -E %error_3
error: Sorry, this is an error

$ echo $?
0
```

The first two invocations work as expected and abort specfile processing, while the third example that calls `%error` via the 4.17+ lua `macros` table does not.

**Comment by pmatilai** [comment]
Looks like a bug, thanks for reporting.

**Comment by pmatilai** [comment]
Fixed in commit 5079c05b001489f1f2e6cbf283c1d72aa83865f9, thanks again for reporting!
